Add wikisource icon

Parthiv-M opened the pull request:
**Issue:** closes #9540 

**Similarweb rank:** 7,516

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
 - Hex value `#006699` was chosen in tune with other wiki projects
 - Icon Preview
![wikisource_icon_preview](https://github.com/simple-icons/simple-icons/assets/57034486/016c1e01-7caf-4e49-b814-5b23194f2a3f)

adamrusted commented on _data/simple-icons.json:
```diff
@@ -14603,6 +14603,15 @@
+            "hex": "006699",
+            "source": "https://commons.wikimedia.org/wiki/File:Wikisource-logo.svg",
+            "license": {
+                "type": "CC-BY-SA-3.0",
+                "url": "https://creativecommons.org/licenses/by-sa/3.0/deed.en"
```
```suggestion
                "type": "CC-BY-SA-3.0"
```

adamrusted reviewed (changes requested):
Thanksfor the contribution @Parthiv-M, though unfortunately it doesn't read particularly well where the lines are. Could you try (feel free to use a bit of artistic liberty with placement) rebuilding with 1px gaps where some of the more major color changes are?

Parthiv-M commented:
Hello @adamrusted, thanks for the review. I have made the changes and updated the preview above as well.

PeterShaggyNoble reviewed (changes requested):
Thanks for your work on this one, @Parthiv-M 👍🏻 However, I'm seeing [some differences](https://github.com/simple-icons/simple-icons/assets/15157491/86fd528a-4e0e-4379-8e58-a52100585d8f) when I overlay your version (in red) with the original.

I wonder, though, if by attempting to retain all the detail of the original is the right way to go in this case and if a more simplified approach might not be a better treatment that more faithfully represents the original in monochrome. Something along the lines, but perhaps not _exactly_ like the below. What do you think?

![](https://github.com/simple-icons/simple-icons/assets/15157491/3981a2d1-e234-47ef-b720-01f79202d2ff)

Parthiv-M commented:
Alright, that sounds good to me. I'll make the changes soon!

PeterShaggyNoble commented:
Are you still working on this, @Parthiv-M?

Parthiv-M commented:
Hey, yes I am, I was just a bit caught up with some personal issues. I'll make sure to make the changes!

PeterShaggyNoble reviewed (changes requested):
That treatment works for me, @Parthiv-M 👍🏻 However, I am seeing [some significant differences](https://github.com/simple-icons/simple-icons/assets/15157491/c5ea4885-9194-4dda-b7ca-c2426b9a75dd) between your version and the source file.

Parthiv-M commented:
![wikisource_preview](https://github.com/simple-icons/simple-icons/assets/57034486/ee9d51ee-e24a-4166-8806-37e99a2f3255)


Made the following changes @PeterShaggyNoble 
- Fixed non-overlapping of paths as far as I could, looked alright to me in my tests
- Removed the slight gap in between two of the bigger elements

Parthiv-M commented:
Looks like there's a lining error, I'll fix that up

Parthiv-M commented:
Looks like there's a lining error, I'll fix that up
